src/app.controller.ts

```diff
@@ -1,12 +1,18 @@
-import { Controller, Get } from "@nestjs/common";
+import { Controller, Get, Version } from "@nestjs/common";
 import { AppService } from "./app.service";
 
-@Controller()
+@Controller({ path: "/" })
 export class AppController {
   constructor(private readonly appService: AppService) {}
 
   @Get()
   getHello(): string {
     return this.appService.getHello();
   }
+
+  @Version("1")
+  @Get("version-one")
```

Comment:
- this defeats the purpose of versioning if you make a different url for different version
- url should be same but with version prefix like v1/user and v2/user
- Note: in above example both url has same path except version number